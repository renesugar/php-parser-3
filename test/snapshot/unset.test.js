const parser = require('../main');

describe("unset", function() {
  it("simple", function() {
    expect(parser.parseEval('unset($var);')).toMatchSnapshot();
  });
  it("multiple", function() {
    expect(parser.parseEval('unset($var, $var, $var);')).toMatchSnapshot();
  });
});
