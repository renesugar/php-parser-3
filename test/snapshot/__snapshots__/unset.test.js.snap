// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unset multiple 1`] = `
Program {
  "children": Array [
    Unset {
      "kind": "unset",
      "variables": Array [
        Variable {
          "byref": false,
          "curly": false,
          "kind": "variable",
          "name": "var",
        },
        Variable {
          "byref": false,
          "curly": false,
          "kind": "variable",
          "name": "var",
        },
        Variable {
          "byref": false,
          "curly": false,
          "kind": "variable",
          "name": "var",
        },
      ],
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`unset simple 1`] = `
Program {
  "children": Array [
    Unset {
      "kind": "unset",
      "variables": Array [
        Variable {
          "byref": false,
          "curly": false,
          "kind": "variable",
          "name": "var",
        },
      ],
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
